TicketBooking.Data/DbContext/TicketBookingDbContext.cs

```diff
@@ -11,7 +11,15 @@ namespace TicketBooking.Data.DbContext
 {
     public class TicketBookingDbContext : IdentityDbContext<ApplicationUser>
     {
+        public static TicketBookingDbContext Create()
+        {
```

Comment:
Why do we need this method?

Reply:
That method is from a guide we follow a long time ago but now it seems not necessary so we already remove it